pkg/apis/apirator/v1alpha1/errors.go

```diff
@@ -0,0 +1,24 @@
+package v1alpha1
+
+import "fmt"
+
+// ingress controller not configured
+type IngressControllerNotFound struct {
```

Comment:
The operator-sdk consider structs declared in package`/pkg/apis/apirator/v1alpha1` as CR, what causes an undesired generation of  'zz_generated.deepcopy.go'